demos/Makefile

```diff
@@ -0,0 +1,13 @@
+SUBDIRS = bio cms evp pkcs12 smime
+
+all: subdirs
+
+subdirs:
+	for dir in $(SUBDIRS); do \
+		$(MAKE) -C $$dir OPENSSL_INCS_LOCATION=-I../../include OPENSSL_LIBS_LOCATION=-L../..; \
```

Comment:
Not all `make` implementations understand `-C`...  might I suggest this instead?

``` make
		( cd $$dir; $(MAKE) OPENSSL_INCS_LOCATION=-I../../include OPENSSL_LIBS_LOCATION=-L../.. ); \
```